src/servers_non_y2/ag_etc_sudoers

```diff
@@ -37,16 +37,16 @@ sub parse_file {
         return 0;
     }
 
-    my $comment = "";
-    my $line    = "";
+    my $line             = "";
+    my $previous_content = "";
 
     while (<INFILE>) {
         chomp;
         $line .= $_;
 
-        # The line is a comment
+        # The line is empty, a comment, or a directive
```

Comment:
Please write example of directives like `#includedir /etc/sudoers.d` . So it is clear what directive here means and how it looks.

BTW this change in general is work-around. So please specify that for now we ignore that directives and agent does not understand it, so it is processed same as comments.

Reply:
> BTW this change in general is work-around.

Sadly, that's true. But, as discussed yesterday in IRC, I think it doesn't pay off to rewrite this agent since the plan is to use a CFA model from SLE-15 on.

Reply:
@jreidinger 

Notes added, thanks.

BTW, I also updated the PR adding a `Warning` section at the end which explains other issues found with the current design of the agent. Please, take a look.